lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9213,6 +9240,11 @@ void ClangImporter::Implementation::swiftify(AbstractFunctionDecl *MappedDecl) {
     }
     for (auto [index, clangParam] : llvm::enumerate(ClangDecl->parameters())) {
       auto clangParamTy = clangParam->getType();
+      UnaliasedInstantiationVisitor visitor;
+      visitor.TraverseType(clangParamTy);
```

Comment:
Instead of doing a separate traversal for each parameter and the return type, can't we just traverse `ClangDecl->getType()` before even instantiating `SwiftifyInfoPrinter`? Would save some work for the cases where we can exit early.